paper/abstract.md

```diff
@@ -0,0 +1,20 @@
+
+
+As higher education decision makers increasingly turn to statistical
+and machine learning algorithms to predict student success, they often
+do so without a clear understanding of how these algorithms work or
+the tradeoffs made when basing decisions upon their predictions. Of
+particular concern are hidden biases inherent in an uncritical use of
+these algorithms that negatively affect already marginalized student
+populations. In this project we use `R` to transparently build a
+number of predictive algorithms and show how decisions based on their
+results might affect already underrepresented student groups in higher
```

Comment:
Same here.  I would leave out already or maybe add "further affect underrepresented student.."

Reply:
Also good with that here; the addition works and I think will keep us under the character count.

Reply:
I changed my mind about the addition, but have removed "already" here in the changes I pushed.